DocumentService/Controllers/DocumentsController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace DocumentService.Controllers
+{
+    [Route("api/")]
+    [ApiController]
+    public class DocumentsController : ControllerBase
+    {
+
+        public DocumentsController()
+        {
+
+        }
+
+        /// <summary>
+        /// Upload a document.
+        /// </summary>
+        /// <param name="CorrelationId">Correlation identifier of the operation.</param>
+        /// <param name="UserName">Azure AD identifier of the user uploading the document.</param>
+        /// <param name="FileName">File name of the document.</param>
+        /// <param name="ShortDescription">Short description of the document.</param>
+        /// <param name="SubmissionMethod">Indicates how the file was submitted to Transport Canada. (“FAX”, “MAIL”, “EMAIL”).</param>
+        /// <param name="FileLanguage">Language of document being uploaded. (1) English, (2) French.</param>
+        /// <param name="FileType">Type of file being uploaded. (“PDF”) pdf file, (“IMG”) Image file, (“TXT”) text file.</param>
```

Comment:
I replaced FileType with FileContentType. See [design document](https://034gc.sharepoint.com/:w:/s/C4CSeafarerProject/EY-zCUWAJklPvBqXnt8KCOMBB4g7sldraYEEqlAZWf69Iw?e=O4cE3L) for explanation.